Firestore/Example/Firestore.xcodeproj/project.pbxproj

```diff
@@ -4726,13 +4754,15 @@
 					"\"${PODS_ROOT}/leveldb-library/include\"",
 				);
 				INFOPLIST_FILE = "App/iOS/Firestore-Info.plist";
+				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				MODULE_NAME = ExampleApp;
 				OTHER_LDFLAGS = (
 					"$(inherited)",
 					"-all_load",
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = "org.cocoapods.demo.${PRODUCT_NAME:rfc1034identifier}";
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SWIFT_VERSION = 5.0;
```

Comment:
Sorry I didn't notice this earlier, but migrating to swift 5 may be premature. Swift 4.2 came with Xcode 10 and AFAIK is still supported.

You'll need to back out the `SWIFT_VERSION` and `LastSwiftMigration` keys in here.

Reply:
Done.